// Copyright (c) 2018 IoTeX
// This is an alpha (internal) release and is not suitable for production. This source code is provided ‘as is’ and no
// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
// License 2.0 that can be found in the LICENSE file.

syntax = "proto3";

package simulator;

// The simulator service definition.
service Simulator {
  rpc Ping (Request) returns (stream Reply) {}
}

// The request message containing the user's name.
message Request {
  int32 playerID = 1;
  int32 senderID = 2;
  int32 messageType = 3;
  string value = 4;
}

// The response message containing the greetings
message Reply {
  int32 playerID = 1;
  int32 senderID = 2;
  int32 messageType = 3;
  string value = 4;
}
